.stylelintrc

```
@@ -1,5 +1,5 @@
 {
-    "extends": "@wordpress/stylelint-config/scss",
+    "extends": "./node_modules/@bcgov/webpack-wordpress/.stylelintrc",
```

Can you not simply do this 
```suggestion
    "extends": "@bcgov/webpack-wordpress/.stylelintrc",
```

---

You can, it turns out. Good catch.